cmake/projects/jsoncpp/hunter.cmake

```diff
@@ -0,0 +1,46 @@
+# cmake/projects/jsoncpp/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
```

Comment:
We need `include(hunter_cmake_args)`. You can run test by this script:
* https://github.com/ruslo/hunter/wiki/dev.hunter.root#jenkinspy